test/CMakeLists.txt

```diff
@@ -29,6 +29,7 @@ endif()
 if(DEFINED ROOT_SOURCE_DIR)
   # Testing using the binary tree
   set(ROOT_root_CMD ${ROOT_BINDIR}/root.exe)
+  include_directories(${ROOT_SOURCE_DIR}/tutorials)
```

Comment:
Why is this addition unconditional?  Why do we need to add something when disabling something else (`PROOF`)?